Docs/source/developers/release.rst

```diff
@@ -0,0 +1,55 @@
+.. _developers-release:
+
+Dependencies & Releases
+=======================
+
+Update WarpX' Core Dependencies
+-------------------------------
+
+WarpX has direct dependencies on AMReX and PICSAR, which we periodically update.
+
+The following scrips automate this workflow, in case one needs a newer commit of AMReX or PICSAR between releases:
```

Comment:
```suggestion
The following scripts automate this workflow, in case one needs a newer commit of AMReX or PICSAR between releases:
```

Reply:
One day I will learn how to type :D Thank you!